### ISSUE TITLE
Doesn't migrate DoubleQuotes rule

### ISSUE THREAD

**Issue by eluchsinger** [angular-eslint/angular-eslint#316]
I don't know, if I'm doing something wrong, but I would like to use Doublequotes in my Typescript.

The `tslint.json` config contains the following entry:

```
"quotemark": [true, "double"],
```

But the resulting eslintrc.json does not contain anything to do with quotemark.  I was expecting the rule to be migrated.

```json
{
  "root": true,
  "ignorePatterns": [
    "projects/**/*"
  ],
  "overrides": [
    {
      "files": [
        "*.ts"
      ],
      "parserOptions": {
        "project": [
          "tsconfig.json",
          "e2e/tsconfig.json"
        ],
        "createDefaultProgram": true
      },
      "extends": [
        "plugin:@angular-eslint/ng-cli-compat",
        "plugin:@angular-eslint/ng-cli-compat--formatting-add-on",
        "plugin:@angular-eslint/template/process-inline-templates"
      ],
      "rules": {
        "@angular-eslint/component-selector": [
          "error",
          {
            "type": "element",
            "prefix": "app",
            "style": "kebab-case"
          }
        ],
        "@angular-eslint/directive-selector": [
          "error",
          {
            "type": "attribute",
            "prefix": "app",
            "style": "camelCase"
          }
        ]
      }
    },
    {
      "files": [
        "*.html"
      ],
      "extends": [
        "plugin:@angular-eslint/template/recommended"
      ],
      "rules": {}
    }
  ]
}

```

**Comment by JamesHenry** [comment]
Thanks @eluchsinger I will change this in v2 (PR is already open) so that it lets the user's configuration for quotes through to the final config.

Right now it just inherits the 'single' + allowTemplateLiterals config set in the configs you are extending from, so you would need to manually add it after `@angular-eslint/directive-selector` if you wanted to customize it to "double"

**Comment by JamesHenry** [comment]
Resolved in the newly published `v2.0.0`, see releases or changelog for more information.
